src/main/java/com/untamedears/JukeAlert/storage/JukeAlertLogger.java

```diff
@@ -1632,11 +1632,19 @@ public boolean updateMutedGroups(UUID uuid, String group2Mute) {
 			return null;
 		} else {
 			// Add the first value
-			ignoringUsers.add(set.getString(1));
+			String[] mutedGroups = set.getString(2).split("\\s+");
+			Set<String> mutedGroupsSet = new HashSet<String>(Arrays.asList(mutedGroups));
```

Comment:
Arrays.asList() appears to be O(1) yea.  new Set( _list_ ) is O(n) -- so at least in that original version you had O(1) + O(n) to get O(1) retrieval (best case). With streamlined, you just have O(1) to get O(n) retrieval, plus no hash computations during set setup, etc. ... largely pedantry at this scale, but always good to keep goal in mind. Spending O(n) for O(1) retrieval makes sense if you do it more then once. In this case where you're setting up a set just to throw it away after a single .contains() -- wouldn't make sense to do the hash setup.